custom/enikshay/management/commands/run_result_summary_display_migration.py

```diff
@@ -0,0 +1,43 @@
+from custom.enikshay.management.commands.utils import BaseEnikshayCaseMigration
+
+DATAMIGRATION_CASE_PROPERTY = 'datamigation_result_summary_display'
+RESULT_SUMMARY_DISPLAY = 'result_summary_display'
+
+
+class Command(BaseEnikshayCaseMigration):
+    case_type = 'test'
+    case_properties_to_update = [
+        RESULT_SUMMARY_DISPLAY,
+    ]
+    datamigration_case_property = DATAMIGRATION_CASE_PROPERTY
+    include_public_cases = True
+    include_private_cases = False
+
+    @staticmethod
+    def get_case_property_updates(test, domain):
+        if (
+            test.get_case_property(DATAMIGRATION_CASE_PROPERTY) == 'yes'
+            or not (
+                test.get_case_property('result_recorded') == 'yes'
+                and not test.get_case_property('test_type_value')
+            )
+        ):
+            return {}
+
+        new_result_display_summary = '\n'.join(filter(None, [
+            {
+                'tb_detected': 'TB Detected',
+                'tb_not_detected': 'TB Not Detected',
+            }.get(test.get_case_property('result')),
```

Comment:
Lets add add a line for test.result_grade (below tb detected / not detected).  Here's the mapping:

negative_not_seen: None/blank
1plus: Result Grade: 1+
2plus: Result Grade: 2+
3plus: Result Grade: 3+
error_invalid_result: Result Grade: Error Invalid Result
0: None/blank
n/a: None/blank
1+: Result Grade: 1+
2+: Result Grade: 2+
3+: Result Grade: 3+

Reply:
There are a lot of different values for ```result_grade``` - sent you a case export on Skype.

How should I handle these other values?

Reply:
@nickpell 

Those might be for private cases?  Is this case export all test cases? I did a review of result_grade for valid private sector cases and I think this is all I got?

Reply:
For those that don't match in our mapping list, can we just use
Result Grade: <test.result_grade>

Reply:
updated